hudi-hadoop-common/src/main/java/org/apache/hudi/storage/hadoop/HadoopStorageConfiguration.java

```diff
@@ -38,6 +38,10 @@ public class HadoopStorageConfiguration extends StorageConfiguration<Configurati
 
   private transient Configuration configuration;
 
+  public HadoopStorageConfiguration() {
+    this(true);
```

Comment:
Using `loadDefaults=true` adds non-trivial overhead of initializing the configuration.  If not necessary (only used in tests), let's remove this constructor and use existing constructor explicitly.